sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/optimizer/NestedColumnAliasingSuite.scala

```diff
@@ -812,6 +812,18 @@ class NestedColumnAliasingSuite extends SchemaPruningTest {
     val expected3 = contact.select($"name").rebalance($"name").select($"name.first").analyze
     comparePlans(optimized3, expected3)
   }
+
+  test("GeneratorNestedColumnAliasing does not pushdown for non-Explode") {
```

Comment:
Can you add JIRA ticket as prefix? E.g. `SPARK-38529: GeneratorNestedColumnAliasing does not pushdown for non-Explode`

Reply:
Done.